dcpy/connectors/esri/arcgis_feature_service.py

```diff
@@ -57,7 +57,7 @@ def query_dataset(dataset: FeatureServer, params: dict) -> dict:
 
 
 def get_dataset(dataset: FeatureServer, crs: int) -> dict:
-    CHUNK_SIZE = 2000
+    CHUNK_SIZE = 100
```

Comment:
@alexrichey any thoughts on if maybe this should just be an override for specific problem datasets? Gut is yes

Reply:
But also simple to just keep it low since performance isn't really an issue

Reply:
Interesting... so does this resolve the issue with the sporadic failures? If so, I'd probably just go the simplest route, and just hardcode like you've done here.

Reply:
It did. NYS GIS Clearinghouse seemed to error out on any given query at exactly 1 minute

Reply:
ah... interesting. Might make sense down the road to add some retry/backoff logic, but this seems great for now